src/Aspire.Dashboard/Configuration/ValidateDashboardOptions.cs

```diff
@@ -144,6 +144,29 @@ public ValidateOptionsResult Validate(string? name, DashboardOptions options)
             }
         }
 
+        if (!options.ReverseProxy.ForwardHeaders)
+        {
+            // TODO: Ensure other properties are not set if ForwardHeaders is false.
+            if (options.ReverseProxy.ForwardLimit != 1)
+            {
+                
+            }
+        }
+
+        // Validate the path base if set to make sure it starts and ends with a forward slash.
+        if (options.PathBase is not null)
+        {
+            if (!options.PathBase.StartsWith('/'))
+            {
+                errorMessages.Add($"{nameof(DashboardOptions.PathBase)} must start with a forward slash.");
```

Comment:
Shouldn't the name here come from a value on `DashboardConfigNames` so the complete path of the option is in the error message?